src/ring_theory/ideal/operations.lean

```diff
@@ -2178,4 +2182,28 @@ lemma quot_quot_equiv_comm_symm :
   (quot_quot_equiv_comm I J).symm = quot_quot_equiv_comm J I :=
 rfl
 
+end
+
+section algebra
+
+@[simp]
+lemma quot_quot_equiv_comm_mk_mk [comm_ring R] (I J : ideal R) (x : R) :
```

Comment:
Does this work for a `semiring`? And same below.

Reply:
This one only works for `comm_ring R` since the definition of `quot_quot_equiv_comm` needs that.
The same holds for `A` in the lemmas below.

Reply:
Ok, it doesn't matter.